doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -271,32 +277,34 @@ Network planning info for resource pool needs to be defined, which should includ
 | DEVICE_MODEL | String | Yes |  |
 | DEVICE_SN | String | Yes |  |
 | BMC_IP | String | Yes |  |
-| SSH_USER | List | Yes |  |
-| SSH_PASSWORD | List | Yes |  |
-| ENABLE_PASSWORD | List | Yes |  |
-| PORT | List | Yes |   |
-| VLAN_ID | List | Yes |   |
-| VLANIF_ADDRESS | List | Yes |   |
+| SSH_USER | String | Yes |  |
+| SSH_PASSWORD | String | Yes |  |
```

Comment:
Do we want to "encourage" end users to capture this type of information in a configuration file?  Should this be SSH public key, or something similar?